packages/wallet/dapp-client/src/utils/index.ts

```diff
@@ -117,8 +117,8 @@ function applyTemplate(template: string, values: Record<string, string>) {
   })
 }
 
-export const getNetwork = (chainId: number) => {
-  const network = allNetworks.find((network) => network.chainId === chainId)
+export const getNetwork = (chainId: Network.ChainId | bigint | number) => {
```

Comment:
might also want to accept Hex | string here too if this is a helper function that could get a network from any type